newIDE/app/src/InstancesEditor/InstancesRenderer/index.js

```diff
@@ -208,8 +316,5 @@ export default class InstancesRenderer {
         this.layersRenderers[i].delete();
       }
     }
-
-    // Finish by the pixi container.
-    this.pixiContainer.destroy();
```

Comment:
Out of curiosity, why is this not needed anymore?

Reply:
Good spot, I'll clean it